doc/pbh/pbh-design.md

```diff
@@ -675,7 +931,11 @@ config pbh rule add 'pbh_table' 'nvgre' \
 --hash 'inner_v6_hash' \
 --packet-action 'SET_ECMP_HASH' \
 --flow-counter 'DISABLED'
-config pbh rule update 'pbh_table' 'nvgre' \
+config pbh rule update field del 'pbh_table' 'nvgre' \
+--ip-protocol
+config pbh rule update field set 'pbh_table' 'nvgre' \
+--ether-type '0x86dd' \
+--ipv6-next-header '0x2f' \
 --flow-counter 'ENABLED'
```

Comment:
Could it be down in single command?
Few options:
```
config pbh rule update 'pbh_table' 'nvgre' --ip-protocol None --ether-type '0x86dd' --ipv6-next-header '0x2f' --flow-counter 'ENABLED'
```

or

```
config pbh rule update 'pbh_table' 'nvgre' --delete-ip-protocol --ether-type '0x86dd' --ipv6-next-header '0x2f' --flow-counter 'ENABLED'
```
or 
```
config pbh rule update 'pbh_table' 'nvgre' --delete-ip-protocol --set-ether-type '0x86dd' --set-ipv6-next-header '0x2f' --set-flow-counter 'ENABLED'
```
They allow a one-shot update when there is interdependency between fields (like "valid only if" constraints on the fields). Also, 1&2 are backward compatible